app/utils/storage-utils.spec.ts

```diff
@@ -47,9 +49,8 @@ describe("StorageUtils", () => {
     });
 });
 
-function assertName(done: any, jobId: string, expected: string) {
+function assertName(jobId: string, expected: string) {
     StorageUtils.getSafeContainerName(jobId).then((containerName: string) => {
```

Comment:
why did you remove done? this is async no? should probably change to an async function all those tests instead of done anyway

Reply:
It's just calling `assertName` at the bottom of the file, which isn't async. Each `assertName` call was calling `done()` at the end, and the entire test would pass as soon as the first `assertName` completed, even if any others were failing.